iati/core/tests/utilities.py

```diff
@@ -63,18 +52,96 @@ def load_as_string(file_path):
     return iati.core.resources.load_as_string(iati.core.resources.get_test_data_path(file_path))
 
 
+DATASET_FOR_ATLEASTONE_RULE_VALID = load_as_dataset('valid_atleastone')
```

Comment:
Should probably remove these constants from this file, replacing situations where they're called with a direct call to `load_as_dataset()`

Reply:
done